Hello_Microverse pull request

FrankMugagga opened the pull request:
1. The README.md file has been updated from the Microverse template
2. Style.css file was imported into the index.html file
3. I added html & css linters configuration
4. Above is the description of the summary of the introduced changes

FrankMugagga committed:
adde indx.html, style.css readme and stylelintrc.json

FrankMugagga committed:
Added the title of the

leonardodiasb commented on README.md:
```diff
@@ -1 +1,97 @@
+ [new_feature_3]
+(back to top)
+
+handshake Contributing
```
- [ ] I see that you added a README file but consider using the [Microverse template](https://github.com/microverseinc/readme-template) for your README and make sure you customize sections like the tile, description, author, and others to match this repository and make it look more professional.

leonardodiasb commented on index.html:
```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+</head>
```
- Unfortunately, the `style.css` is not loading in this HTML file. Kindly import the CSS file to make the style changes work.

leonardodiasb reviewed (changes requested):
# Required Changes ♻️ ⚠️ 🚦 

Hi @FrankMugagga, 

Good job so far!
You still need to work on some issues to go to the next project, but you are almost there!

![mario_ladders](https://user-images.githubusercontent.com/20563252/121369265-4e418880-c901-11eb-848c-1eff06d5a4cf.gif)

## Good points 👍 

 - Correct use of GitHub flow. ✔️

## Aspects to improve ♻️

- [x] Check the comments under the review.
- [ ] The linters configuration is missing in this repository. Please, follow the instructions [in this article](https://github.com/microverseinc/linters-config) and make sure you install the linters and add the linters.yml file to the /github/workflows/ folder to make it run on the PR.
- [ ]  Your Pull Request should have a summary describing the introduced changes in more detail. Kindly edit those to make them look more professional so the code reviewer can understand better what to review.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

FrankMugagga committed:
Update README.md

I added the Microverse template to update my README.md file

FrankMugagga committed:
Update index.html

I added an external style sheet link to the HTML header

FrankMugagga committed:
Create .gitignore

I created a .gitignore file and added a node_module

FrankMugagga committed:
Create .hintrc

I created a .hintrc file

FrankMugagga committed:
Create linters.yml

I created and added the .github/workflows/linter.yml file

binhussen commented on README.md:
```diff
@@ -1 +1,288 @@
+-->
+
+<div align="center">
+  <!-- You are encouraged to replace this logo with your own! Otherwise you can also remove it. -->
+  <img src="murple_logo.png" alt="logo" width="140"  height="auto" />
+  <br/>
```
- kindly delete the logo and replace it with your project logo or make it empty.👌

FrankMugagga replied:
this has been deleted

binhussen commented on README.md:
```diff
@@ -1 +1,288 @@
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 [Hello_Microverse] <a name="about-project"></a>
+
+> Describe your project in 1 or 2 sentences.
+
```
- Kindly Replace it with your project description👌👌

FrankMugagga replied:
Ihave replaced it

binhussen commented on README.md:
```diff
@@ -1 +1,288 @@
+
+> Describe the tech stack and include only the relevant sections that apply to your project.
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://reactjs.org/">React.js</a></li>
+  </ul>
+</details>
+
+<details>
+  <summary>Server</summary>
+  <ul>
+    <li><a href="https://expressjs.com/">Express.js</a></li>
+  </ul>
+</details>
+
+<details>
+<summary>Database</summary>
+  <ul>
+    <li><a href="https://www.postgresql.org/">PostgreSQL</a></li>
+  </ul>
+</details>
```
- Kindly add the technology you used and remove unused technology.👌👌 to make your project distinguish by seeing the readme👌

binhussen commented on README.md:
```diff
@@ -1 +1,288 @@
+
+> Mention all of the collaborators of this project.
+
+👤 **Author1**
+-** FRANK MUGAGGA**
+- GitHub: [@githubhandle](https://github.com/githubhandle)
+- Twitter: [@twitterhandle](https://twitter.com/twitterhandle)
+- LinkedIn: [LinkedIn](https://linkedin.com/in/linkedinhandle)
+
+👤 **Author2**
+
+- GitHub: [@githubhandle](https://github.com/githubhandle)
+- Twitter: [@twitterhandle](https://twitter.com/twitterhandle)
+- LinkedIn: [LinkedIn](https://linkedin.com/in/linkedinhandle)
```
- Kindly replace the author info with yours and delete the second author to make your contact easily accessible.👌

binhussen commented on .gitignore:
```diff
@@ -0,0 +1 @@
+node_modules/
```
- [Optional] Kindly create the test.md as the requirement and include it in git ignore 👌👌

binhussen reviewed (changes requested):
Hi @FrankMugagga , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<img src="https://media0.giphy.com/media/UdaSUYClr92cjWpeVx/giphy.gif"/>

## Highlights

- All linter checks passed ✔️
- Descriptive pull request ✔️
- Used GitHub flow ✔️

## Required Changes ♻️
- Kindly update the readme with your info 👌👌

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

FrankMugagga commented:
Hello,
Do I still need to send a review request. it will be my last though

regards
frank

On Wed, Jan 11, 2023 at 10:28 PM Mohammed Hussen ***@***.***>
wrote:

> ***@***.**** requested changes on this pull request.
>
> Hi @FrankMugagga <https://github.com/FrankMugagga> ,
>
> Good job so far!
> There are some issues that you still need to work on to go to the next
> project but you are almost there!
>
> <https://camo.githubusercontent.com/679757134a8fee76cfe1941f9f3ecb0e0cf56dd02da4498268ce31f0dacdfca3/68747470733a2f2f6d65646961302e67697068792e636f6d2f6d656469612f556461535559436c723932636a57706556782f67697068792e676966>
> Highlights
>
>    - All linter checks passed ✔️
>    - Descriptive pull request ✔️
>    - Used GitHub flow ✔️
>
> Required Changes ♻️
>
>    - Kindly update the readme with your info 👌👌
>
> Optional suggestions
>
> *Every comment with the [OPTIONAL] prefix is not crucial enough to stop
> the approval of this PR. However, I strongly recommend you to take them
> into account as they can make your code better.*
>
> Cheers and Happy coding!👏👏👏
>
> Feel free to leave any questions or comments in the PR thread if something
> is not 100% clear.
> Please, remember to tag me in your question so I can receive the
> notification.
>
> *Please, do not open a new Pull Request for re-reviews. You should use the
> same Pull Request submitted for the first review, either valid or invalid
> unless it is requested otherwise.*
> ------------------------------
>
> *As described in the Code reviews limits policy
> <https://microverse.zendesk.com/hc/en-us/articles/1500004088561> you have a
> limited number of reviews per project (check the exact number in your
> Dashboard). If you think that the code review was not fair, you can request
> a second opinion using this form <https://airtable.com/shrQAqnBwek5a0O0s>.*
> ------------------------------
>
> In README.md
> <https://github.com/FrankMugagga/Hello_Microverse/pull/1#discussion_r1067369712>
> :
>
> > +  <!-- You are encouraged to replace this logo with your own! Otherwise you can also remove it. -->
>
> +  <img src="murple_logo.png" alt="logo" width="140"  height="auto" />
>
> +  <br/>
>
>
>
>    - kindly delete the logo and replace it with your project logo or make
>    it empty.👌
>
> ------------------------------
>
> In README.md
> <https://github.com/FrankMugagga/Hello_Microverse/pull/1#discussion_r1067370615>
> :
>
> > +# 📖 [Hello_Microverse] <a name="about-project"></a>
>
> +
>
> +> Describe your project in 1 or 2 sentences.
>
> +
>
>
>
>    - Kindly Replace it with your project description👌👌
>
> ------------------------------
>
> In README.md
> <https://github.com/FrankMugagga/Hello_Microverse/pull/1#discussion_r1067372529>
> :
>
> > +<details>
>
> +  <summary>Client</summary>
>
> +  <ul>
>
> +    <li><a href="https://reactjs.org/">React.js</a></li>
>
> +  </ul>
>
> +</details>
>
> +
>
> +<details>
>
> +  <summary>Server</summary>
>
> +  <ul>
>
> +    <li><a href="https://expressjs.com/">Express.js</a></li>
>
> +  </ul>
>
> +</details>
>
> +
>
> +<details>
>
> +<summary>Database</summary>
>
> +  <ul>
>
> +    <li><a href="https://www.postgresql.org/">PostgreSQL</a></li>
>
> +  </ul>
>
> +</details>
>
>
>
>    - Kindly add the technology you used and remove unused technology.👌👌
>    to make your project distinguish by seeing the readme👌
>
> ------------------------------
>
> In README.md
> <https://github.com/FrankMugagga/Hello_Microverse/pull/1#discussion_r1067377651>
> :
>
> > +👤 **Author1**
>
> +-** FRANK MUGAGGA**
>
> +- GitHub: ***@***.***(https://github.com/githubhandle)
>
> +- Twitter: ***@***.***(https://twitter.com/twitterhandle)
>
> +- LinkedIn: [LinkedIn](https://linkedin.com/in/linkedinhandle)
>
> +
>
> +👤 **Author2**
>
> +
>
> +- GitHub: ***@***.***(https://github.com/githubhandle)
>
> +- Twitter: ***@***.***(https://twitter.com/twitterhandle)
>
> +- LinkedIn: [LinkedIn](https://linkedin.com/in/linkedinhandle)
>
>
>
>    - Kindly replace the author info with yours and delete the second
>    author to make your contact easily accessible.👌
>
> ------------------------------
>
> In .gitignore
> <https://github.com/FrankMugagga/Hello_Microverse/pull/1#discussion_r1067379241>
> :
>
> > @@ -0,0 +1 @@
>
> +node_modules/
>
>
>
>    - [Optional] Kindly create the test.md as the requirement and include
>    it in git ignore 👌👌
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/FrankMugagga/Hello_Microverse/pull/1#pullrequestreview-1244476979>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/A4HU4OMEDXRBFQSBWH6I53DWR4CVHANCNFSM6AAAAAATX5UP5U>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>

FrankMugagga committed:
Update README.md

I deleted the logo
I replaced README.md with the repository  name
I removed the technology I didn't use and replaced it with that that I used
I replaced the author info with mine and removed the second

FrankMugagga committed:
Create test.md

I added a test.md file

FrankMugagga committed:
Update .gitignore

I have added test.md in .gitignore to be ignored

joskalenda reviewed (approved):
| STATUS: APPROVED ✅ |
|------------------------|
| Hi @FrankMugagga , 🥰|
| Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: Congratulations! 🎉  |
| <img src="https://media2.giphy.com/media/2NpKgBuUZtXpu/giphy.gif"/> |

### Highlights
✅ ... Green linters
✅ ... Great communication with the review

---------
### Optional suggestions

> _Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

### Cheers and Happy coding!👏👏👏
-------
> Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me(@joskalenda) in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

FrankMugagga commented:
Hello Jos,
Thank you very much
regards
frank

On Wed, Jan 11, 2023 at 11:35 PM Jos Kalenda ***@***.***>
wrote:

> ***@***.**** approved this pull request.
> STATUS: APPROVED ✅
> Hi @FrankMugagga <https://github.com/FrankMugagga> , 🥰
> Your project is complete! There is nothing else to say other than... it's
> time to merge it [image: :shipit:] Congratulations! 🎉
>
> <https://camo.githubusercontent.com/51746859d842dcf7cf581bfd25e8426ce89edbb4b0105080e43e2379340bcfd4/68747470733a2f2f6d65646961322e67697068792e636f6d2f6d656469612f324e704b674275555a745870752f67697068792e676966>
> Highlights
>
> ✅ ... Green linters
> ✅ ... Great communication with the review
> ------------------------------
> Optional suggestions
>
> *Every comment with the [OPTIONAL] prefix won't stop the approval of this
> PR. However, I strongly recommend you take them into account as they can
> make your code better. Some of them were simply missed by the previous
> reviewer and addressing them will really improve your application.*
>
> Cheers and Happy coding!👏👏👏
> ------------------------------
>
> Feel free to leave any questions or comments in the PR thread if something
> is not 100% clear.
> Please, remember to tag ***@***.*** <https://github.com/joskalenda>)
> in your question so I can receive the notification.
>
> ------------------------------
>
> *As described in the Code reviews limits policy
> <https://microverse.zendesk.com/hc/en-us/articles/1500004088561> you have a
> limited number of reviews per project (check the exact number in your
> Dashboard). If you think that the code review was not fair, you can request
> a second opinion using this form <https://airtable.com/shrQAqnBwek5a0O0s>.*
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/FrankMugagga/Hello_Microverse/pull/1#pullrequestreview-1244576565>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/A4HU4OL5HDHKHXSA7UK66KTWR4KQFANCNFSM6AAAAAATX5UP5U>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>
